### PR TITLE
Same shebang for repro and buildinfo

### PR DESCRIPTION
**Pull request by andrin-geiger**
buildinfo already uses `#!/usr/bin/env bash`

### CODE REVIEW

File: repro.in [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```
Comment (jelly) [inline repro.in]: Why is this needed? And with what does it make us more "compatible"?

Comment (andrin-geiger) [reply repro.in]: One of the goals of this project is to be distribution independent. NixOS for example doesn't have a bash binary in `/bin`.

Comment (Foxboron) [reply repro.in]: Nix is sufficiently weird that I'm not really going to care about them unless someone is intentionally trying to run `repro` on NixOS.

Comment (Foxboron) [inline repro.in]:
```suggestion
#!/usr/bin/bash
```

Do the same with `buildinfo` please.

Comment (jelly) [reply repro.in]: Afaik, NixOS patches shebangs and asks their contributors specifically not to make PR's to every piece of software ever existed to solve their own invented problem. Python does the same imo `/usr/bin/python` not `/usr/bin/env python`.

Someone is welcome to package repro for NixOS and then patch it there.

Also plain using `env` causes problems as https://discourse.nixos.org/t/how-do-you-run-a-bash-script/10141/8

Comment (andrin-geiger) [reply repro.in]: I understand that using the same shebang in both script makes sense.
But why `#!/usr/bin/bash` over `#!/bin/bash`?
Some other scripts like `makepkg` or `makerepropkg` are also using `#!/bin/bash`

Comment (Foxboron) [reply repro.in]: `makepkg` predates this script and for all I know runs on systems without usrmerge.

`repro` can only run on `systemd` distros and those should have implemented usrmerge by now.

Comment (kpcyrd) [reply repro.in]: I'm not sure about Debian's usrmerge status, they ship their bash at `/bin/bash` and `/usr/bin/bash` is not necessarily available, however on usrmerge systems like Arch Linux `/bin` is symlinked to `/usr/bin`, so `/bin/bash` would work there too.

**Review by Foxboron — APPROVED** [review]
